kk/tests/test_comment.py

```diff
@@ -63,14 +74,133 @@ def test_55_add_comment_to_hearing(self, default_hearing):
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id, 'comments'))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        # get first returned comment
+        comment = data[0]
+
+        assert 'content' in comment
+        assert 'created_at' in comment
+        assert 'votes' in comment
+        assert 'created_by' in comment
+
+    def test_54_list_all_comments_added_to_hearing_check_content(self, default_hearing):
+        self.user_login()
+        self.create_default_comments(default_hearing)
+
+        # list all comments
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id, 'comments'))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        contents = [c['content'] for c in data]
+
+        assert self.default_content in contents
+        assert self.red_content in contents
+        assert self.green_content in contents
+
+    def test_54_list_all_comments_added_to_hearing_check_votes(self, default_hearing):
+        self.user_login()
+        self.create_default_comments(default_hearing)
+
+        # list all comments
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id, 'comments'))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        for comment in data:
+            assert comment['votes'] == 0
+
+    def test_54_list_all_comments_added_to_hearing_check_created_by(self, default_hearing):
+        self.user_login()
+        self.create_default_comments(default_hearing)
+
+        # list all comments
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id, 'comments'))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        for comment in data:
+            assert comment['created_by'] == self.username
+
+    def test_54_list_all_comments_added_to_hearing_check_created_at(self, default_hearing):
+        self.user_login()
+        self.create_default_comments(default_hearing)
+
+        # list all comments
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id, 'comments'))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        for comment in data:
+            assert datetime.datetime.now().strftime('%Y-%m-%dT%H:%M') in comment['created_at']
+
+    def test_54_get_hearing_with_comments_check_amount_of_comments(self, default_hearing):
+        self.user_login()
+        self.create_default_comments(default_hearing)
+
+        # list all comments
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id))
+        assert response.status_code == 200
+
+        data = self.get_data_from_response(response)
+        assert 'comments' in data
+
+        assert len(data['comments']) == 3
```

Comment:
Should actually probably just add a post-save signal for `HearingComment`s to do that.